include/boost/json/value.hpp

```diff
@@ -2832,6 +2832,65 @@ struct key_value_pair
 
 //----------------------------------------------------------
 
+#ifdef BOOST_JSON_DOCS
+
+/** Tuple-like element access.
+    
+    This overload permits the key and value
+    of a `key_value_pair` to be accessed
```

Comment:
```
This function returns members of a `@ref key_value_pair` by index.
```

Reply:
Eh... I'd rather say "key and value". Members is too broad, and there's only two that we provide access to.

Reply:
OK

Reply:
The reason I worded it that way is to avoid repetition of the words "key" and "value" but we can go with your wording too.